clients/venice-thin-client/src/main/java/com/linkedin/venice/client/store/AvroGenericStoreClient.java

```diff
@@ -81,22 +125,14 @@ default CompletableFuture<VeniceResponseMap<K, V>> streamingBatchGet(Set<K> keys
    * @param callback
    * @throws VeniceClientException
    */
-  default void streamingBatchGet(Set<K> keys, StreamingCallback<K, V> callback) throws VeniceClientException {
-    throw new VeniceClientException(
-        "Please use CachingVeniceStoreClientFactory#getAndStartAvroGenericStoreClient() "
-            + "or VeniceGenericStoreClientFactory#createInstance() to generate a Venice avro generic client");
-  }
+  void streamingBatchGet(Set<K> keys, StreamingCallback<K, V> callback) throws VeniceClientException;
 
   /**
-   * This experimental feature is subject to backwards-incompatible changes and may even be removed in the future.
-   * @return
+   * This API allows performing transformations on the values associated with the given set of keys. Check out
```

Comment:
`transformation` -> `computation`? 
Projection, and arithmetic ops are not transformation from my understanding.

Reply:
Transformation would be a superset of both projections and computations - so, seems right to use here. Let me know if you disagree

Reply:
To me, `transform` is to convert some record from one format to another, and computation are not really transformation since for projection, it will stay as it is and for arithmetic ops, it will execute some vector operations against two vectors and get either a float back and a vector back.
`compute` term has been communicated with customers for quite some time, and `transformation` might cause some confusion.

If you think `compute` is not enough to cover `project`, I would prefer we put `project` and `compute` in the javadoc.

Reply:
I updated the javadoc to include both `projection` and `aggregation`. Do we need to call out `compute` explicitly? Does the new javadoc look okay?

Reply:
`aggregation` sounds a little unclear to me.
The supported compute operators:
1. projection.
2. Vector arithmetic.
3. Count.

`Read Compute` was used as a general term to cover all of them and that is the reason we named the api as `compute`.
Maybe we can continue to use `compute` term and expand it in the javadoc to describe the supported compute operations to make it more comprehensive.
What do you think?

Reply:
Sure. That works too

Reply:
I've updated the javadoc to reflect this. How does this look:
```
  /**
   * This API allows performing transformations (projection, vector arithmetic and aggregations like count) on the
   * values associated with the given set of keys. Check out {@link ComputeRequestBuilder} for details on the available
   * operations.
   * @see ComputeRequestBuilder
   */
```